.github/workflows/build-packages.yml

```
@@ -41,60 +41,65 @@ jobs:
       with:
         egress-policy: audit
 
     - name: Checkout
-      uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+      uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6.0.2
       with:
         path: temp-flex
 
+    - name: Getting build dependencies
+      id: get_dependencies
+      run: |
```

actionlint (1.7.12)

[error] string should not be empty

(syntax-check)